common/snapshots.py

```diff
@@ -844,15 +844,12 @@ def backup(self, force=False):
                                 'Cannot start snapshot yet: target directory '
                                 'not accessible. Will retry each second in '
                                 'the next 30 seconds. Please wait.')
-
-                            counter = 0
-                            for counter in range(0, 30):
+                            for _ in range(30):
```

Comment:
Stupid me! Even if I explained it to you I made the same mistake. That is why TravisCI failes.

```suggestion
                            for _idx in range(30):
```

Reply:
Don't worry!! I noticed and just changed it